openjdk/ProblemList_openjdk11-openj9.txt

```diff
@@ -133,7 +133,7 @@ sun/management/HotspotRuntimeMBean/GetSafepointCount.java	https://github.com/Ado
 sun/management/HotspotRuntimeMBean/GetSafepointSyncTime.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/932 generic-all
 sun/management/HotspotRuntimeMBean/GetTotalSafepointTime.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/932 generic-all
 sun/management/HotspotThreadMBean/GetInternalThreads.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/932 generic-all
-
+java/lang/management/MemoryMXBean/ResetPeakMemoryUsage.java https://github.ibm.com/runtimes/openj9-openjdk-jdk11-zos/issues/454 generic-all
```

Comment:
@psoujany - please use https://github.com/AdoptOpenJDK/openjdk-tests/issues/1297 instead of the https://github.ibm.com/runtimes/openj9-openjdk-jdk11-zos/issues/454 internal issue